packages/manager/apps/container/src/container/legacy/server-sidebar/universe/HostedPrivateCloudSidebar.tsx

```diff
@@ -189,27 +189,24 @@ export default function HostedPrivateCloudSidebar() {
           feature['vrack-services'] && {
             id: 'dedicated-vrackservices',
             label: t('sidebar_vrack_services'),
-            icon: getIcon('oui-icon oui-icon-vRack-services_concept'),// TODO Add new icon
-            href: navigation.getURL('vrack-services', '#/'),
+            icon: getIcon('oui-icon oui-icon-vRack-services_concept'),
             routeMatcher: new RegExp('^/vrack-services'),
-            // async loader() {
-            //   const services = await loadServices('/vrackServices/resource');//TODO Needs to relook with apiv2 api
-            //   return [
-            //     {
-            //       id: 'vrack_services-all',
-            //       label: t('sidebar_service_all'),
-            //       href: navigation.getURL('dedicated', '#/vrack-services'),
-            //       ignoreSearch: true,
-            //     },
-            //     ...services.map((service) => ({
-            //       ...service,
-            //       href: navigation.getURL(
-            //         'dedicated',
-            //         `#/vrack-services/${service.currentState.vrackId}`,
-            //       ),
-            //     })),
-            //   ];
-            // },
+            async loader() {
+              const items = await loadServicesV2({
+                applicationId: 'vrack-services',
+                path: '/vrackServices/resource'
+              });
+
+              return [
+                {
+                  id: 'vrack_services-all',
+                  label: t('sidebar_service_all'),
+                  href: navigation.getURL('vrack-services', '#/'),
+                  ignoreSearch: true,
+                },
+                ...items
```

Comment:
**nitpick:** A generic translate key is used. You must add a new key for vrack-services product

Reply:
done thank you